### DISCUSSION TITLE
Ring of the seven curses & Enigmatic amulet

### DISCUSSION THREAD

**Discussion by Reniel80** [Aizistral-Studios/Enigmatic-Legacy#255] [Ideas]
It's too easy to have them in the inventory at the beginning of every new world. It's a problem if you equip the Ring of the seven curses... imagine you are new to minecraft or you are a kid who is learning english just for those who are curious to see what it does ..... the only solution to that problem is to delete your world (imagine the big pain after all the hard work for the beginners)

1- Could it be possible that the ring of the seven curses self-destroy when you die if it is equipped?

(It is preferable that it self-destroy along with all the items in your inventory and also all you have equipped)

2- Could it be possible, not to start with the Ring of the seven curses but to find them in a loot or chest?

3 - Start a simple quest to obtain the enigmatic amulet.

**Comment by Aizistral** [comment]
You can disable both Enigmatic Amulet and Ring of the Seven Curses from being granted to player via respective options in Accessibility Config section of `enigmaticlegacy-common.omniconf` config file. This should be enough for modpack developers who might not want those items to be accessible from the start of the game in their modpack.

However, I am not willing to change how things are by default. Regarding Ring of the Seven Curses, it essentially presents an entire new gamemode, and to me it is crucial that player can enable such gamemode for themselves from day one; regarding Enigmatic Amulet, I believe abilities it grants are nothing close to overpowered, and later on it ends up being outcompeted by more powerful accessories wanting that same charm slot, only remaining useful for it's gravestone-like ability.

And, yeah, I understand that a lot of players can end up equipping Ring of the Seven Curses without fully understanding the consequences. I have undertaken every effort to prevent such cases, ensuring that the ring cannot be equipped by simply right-clicking, and supplying it with extensive tooltip with clear and understandable warnings. This might not be enough for some, but, let's be honest - even if I made a pop-up screen with big red writings reading "YOU ARE ABOUT TO CONDEMN YOURSELF TO MUCH SUFFERING THERE BOI" upon trying to equip the ring, it would still not be enough for some.

**Reply by JaegerPhantom** [reply]
You should just add to the FAQ that the ring will not be changed and that ignorance will be punished. You will never be able to make people realize it.

**Comment by MabaS095** [comment]
hello i know how remove the ring of the seven curse without losing your world even nothing you have just install NBTexplorer and find your world and your player data find ForgeCaps and look curios inventory find the right slot cursed ring and delete the data in Stacks folder and enjoy if you need help 
my discord: Ma_baS
good morning
